app/src/main/java/com/example/nasaapp/ui/search/SearchAdapter.kt

```diff
@@ -1,12 +1,17 @@
 package com.example.nasaapp.ui.search
 
+import android.os.Bundle
 import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.FragmentActivity
 import androidx.recyclerview.widget.RecyclerView
+import com.example.nasaapp.R
 import com.example.nasaapp.domain.model.SearchItem
 
 class SearchAdapter(private val items: List<SearchItem>) :
```

Comment:
укажи параметром лямбду
```suggestion
class SearchAdapter(private val items: List<SearchItem>, onItemClick: () -> Unit) :
```